### PR TITLE
[WIP] Show Author and Maintainer in Project detail view

### PR DESCRIPTION
**Pull request by timofurrer**
Refs #1060 

@dstufft I need some input regarding the implementation of email validation. How do you want to have this implemented? Should I use a self written primitive regex? Or a package like https://pypi.python.org/pypi/validate_email ? Something else?

### CODE REVIEW

**Comment by sigmavirus24** [comment]
Please don't write a regular expression. They will never work in your favor: http://haacked.com/archive/2007/08/21/i-knew-how-to-validate-an-email-address-until-i.aspx/

`validate_email` is LGPL licensed and I'm not sure what the warehouse developers will think of that. I'm fairly certain they could use it, but IANAL and they should check with one first. =)

**Comment by timofurrer** [comment]
> Please don't write a regular expression. They will never work in your favor: http://haacked.com/archive/2007/08/21/i-knew-how-to-validate-an-email-address-until-i.aspx/

Nice article. Thanks :beers:

**Comment by sigmavirus24** [comment]
You're welcome! :cake:

File: warehouse/templates/packaging/detail.html [hunk 1]
```diff
@@ -126,16 +126,19 @@ <h3 class="sidebar-section__title">Navigation</h3>
...
-
         {% if release.has_meta %}
         <div class="sidebar-section">
           <h3 class="sidebar-section__title">Meta</h3>
+          {% if release.author_email %}
+            <p>Author: <a href="mailto:{{ release.author_email }}">{{ release.author or release.author_email }}</a></p>
+          {% elif release.author %}
+            <p>Author: {{ release.author }}</p>
+          {% endif %}
+          {% if release.maintainer_email %}
```
Comment (berkerpeksag) [inline warehouse/templates/packaging/detail.html]: If I read https://github.com/pypa/warehouse/issues/1060#issuecomment-229860780 correctly, this is not needed.

Comment (timofurrer) [inline warehouse/templates/packaging/detail.html]: What about that one? https://github.com/pypa/warehouse/issues/1060#issuecomment-229932169

Comment (di) [inline warehouse/templates/packaging/detail.html]: I think @timofurrer has the correct interpretation here.

**Comment by di** [comment]
Hey @timofurrer, this looks great to me. Can you resolve the merge conflicts?

**Pull request closed without merging**